src/session/session_compact.c

```diff
@@ -175,6 +175,22 @@ __compact_handle_append(WT_SESSION_IMPL *session, const char *cfg[])
     return (0);
 }
 
+/*
+ * __compact_timing_stress --
+ *     Optionally add a delay to the compact for debug purposes.
+ */
+static void
+__compact_timing_stress(WT_SESSION_IMPL *session, uint64_t flag, struct timespec *tsp)
```

Comment:
I feel like it's not a right thing to pass the flag you want to test for. If `__compact_timing_stress()` is only called from this file then a better approach is to not pass the flag.

Reply:
If we remove  `flag` from the arguments list then we don't need `tsp` either.

Reply:
Fixed in 28562ccdda07fcdc52f0b92ce38b9138ad5870dc